deploy/kubedirector/crd-settings.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kubedirectorsettingss.kubedirector.bluedata.io
```

Comment:
kubedirectorsettings

Reply:
it has to be plural here

Reply:
ok.

Reply:
I think he means you have too many ssssses.

Reply:
Oh I see, because the singular ends with an s. Hmm. Can we just make the plural the same as the singular, does k8s care?

Reply:
It has to be.a plural. k8s will fail otherwise. We can change to config though.